flocker/node/script.py

```diff
@@ -173,7 +199,10 @@ def main(self, reactor, options, volume_service):
         deployer = P2PManifestationDeployer(ip, volume_service,
                                             node_uuid=node_uuid)
         loop = AgentLoopService(reactor=reactor, deployer=deployer,
-                                host=host, port=port)
+                                host=host, port=port,
+                                context_factory=_context_factory(
+                                    options["agent-config"].parent(),
+                                    host, port))
```

Comment:
I guess `ZFSAgentScript` is going to be removed soon so this duplication isn't a big deal.